deploy/osd-delete-backplane-serviceaccounts/00-delete-backplane-serviceaccounts.rbac.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: backplane
```

Comment:
This namespaces does not exists on all OSD clusters, and we should create one first, per the name, we should have a "openshift-" prefix.